src/compiler/transformers/declarations.ts

```diff
@@ -77,9 +77,10 @@ namespace ts {
             moduleResolverHost: host,
             trackReferencedAmbientModule,
             trackExternalModuleSymbolOfImportTypeNode,
-            reportNonlocalAugmentation
+            reportNonlocalAugmentation,
+            reportNonSerializableProperty
         };
-        let errorNameNode: DeclarationName | undefined;
+        let errorNameNode: DeclarationName | undefined; // What's this??

```

Comment:
It's the closest `DeclarationName` the declaration emitter has recurred into. Might wanna remove this comment. :3